telemetry/attrs.go

```diff
@@ -33,8 +33,16 @@ const (
 	UIInternalAttr = "dagger.io/ui.internal"
 
 	// Hide child spans by default.
+	//
+	// Encapsulated child spans may typically be revealed if the parent span errors.
 	UIEncapsulateAttr = "dagger.io/ui.encapsulate"
 
+	// Hide span by default.
+	//
+	// This is functionally the same as UIEncapsulateAttr, but is instead set
+	// on a child instead of a parent.
+	UIEncapsulatedAttr = "dagger.io/ui.encapsulated"
```

Comment:
Cool idea, but is this significantly different from `Internal()`? :thinking:

Reply:
Yes! So with Internal, if it fails, it always shows.

These requests will sometimes fail, e.g. with a 401 if they need authentication (this happens pretty often). Encapsulated lets us hide this unless the failure propagates to the parent, and the whole pull fails.